best-practices/database/mongodb/PROTECT.md

```diff
@@ -2,117 +2,116 @@
 
 ### Protect your MongoDB
 
-1. Lock down access to your MongoDB instance's port 27017  
-        _(reference: [guide](https://medium.com/@matteocontrini/how-to-setup-auth-in-mongodb-3-0-properly-86b60aeef7e8#.2tbt9or6y))_  
-        * _27017 is MongoDB's default listening port and as such is most vulnerable to attacks._ 
```

Comment:
I only have ESLint extension installed on VSCode. I am not sure why some of the code is lit up red as if I edited those lines. I only updated the spelling of github to `GitHub`.

Reply:
Looks like it might have gotten hit by an auto formatter like Prettify.  I find that it's helpful to use a visual Git tool to verify contents of the changed files and only stage the ones you intended to edit.

Reply:
After taking look it seems that indeed prettier extension added a `\` to line starting with `*`, which is the equivalent to an asterisk. 
It also changed all the asterisk used for bullets to `-`, which is also equivalent to indented bullets. 
Previewing the document itself doesn't show any change.
Do we want to change this back to `*` bullets?

Reply:
Is there a benefit to using the `-`  or other markdown?

Reply:
[Markdown Reference](https://wordpress.com/support/markdown-quick-reference/) used does not indicate any pros or cons for using `*` or `-`.